apps/api/src/app/workflows-v2/dtos/controls-metadata.dto.ts

```diff
@@ -6,7 +6,7 @@ import { UiSchema } from './ui-schema.dto';
 export class ControlsMetadataDto {
   @ApiPropertyOptional({
     description: 'JSON Schema for data',
-    oneOf: [{ $ref: '#/components/schemas/JSONSchemaDto' }],
+    type: () => Object,
```

Comment:
This was causing an extremly huge Open API Spec. 300k lines of code due to the nested behavior of it.